manifests/c/charm/glow/1.5.1/charm.glow.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: charm.glow
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: charm
+PublisherUrl: https://charm.sh/
+PublisherSupportUrl: https://charm.sh/chat/
+Author: Charmbracelet, Inc
+PackageName: Glow
+PackageUrl: https://github.com/charmbracelet/glow
+License: MIT
+LicenseUrl: https://github.com/charmbracelet/glow/blob/master/LICENSE
+Copyright: Copyright (c) 2019-2023 Charmbracelet, Inc
+ShortDescription: Render markdown on the CLI, with pizzazz!

```

Comment:
```suggestion
Publisher: charmbracelet
PublisherUrl: https://charm.sh
PublisherSupportUrl: https://charm.sh/chat/
Author: Charmbracelet, Inc
PackageName: glow
PackageUrl: https://github.com/charmbracelet/glow
License: MIT
LicenseUrl: https://github.com/charmbracelet/glow/blob/master/LICENSE
Copyright: Copyright (c) 2019-2023 Charmbracelet, Inc
ShortDescription: Render markdown on the CLI, with pizzazz!
Description: |-
  Glow is a terminal based markdown reader designed from the ground up to bring out the beauty—and power—of the CLI.
  Use it to discover markdown files, read documentation directly on the command line and stash markdown files to your own private collection, so you can read them anywhere.
  Glow will find local markdown files in subdirectories or a local Git repository.
Moniker: glow
Tags:
- charmbracelet
- cli
- excitement
- markdown
ReleaseNotes: |-
  What's Changed
  -  fix: rm emoji from package description by @caarlos0 in #453
  -  fix: improve editor handling by @caarlos0 in #449
  -  fix: do not use deprecated func by @caarlos0 in #451
  -  docs: fix typos by @kianmeng in #471
  -  chore(deps): bump actions/setup-go from 3 to 4 by @dependabot in #478
  -  fix: lazily init UI by @muesli in #494
ReleaseNotesUrl: https://github.com/charmbracelet/glow/releases/tag/v1.5.1
```

Reply:
These notes are imported from the [other version](https://github.com/microsoft/winget-pkgs/pull/107372/files).